packages/split-ssh/src/split-ssh-formula/state/policy/files/qubes.SSHAgent

```diff
@@ -1 +1,5 @@
-ssh-client ssh-vault ask
+{% for vault in pillar.split-ssh-vault %}
+{% for client in pillar.split-ssh-client %}
+{{ client.name }} {{ vault.name }} ask
+{% endfor %}
+{% endfor %}
```

Comment:
All clients can ask any of the vaults for SSH keys.

The default behavior (`ask` or `allow`) could be configured, but I don't see the need for now.